### PR TITLE
Fix the description of paramter '--topics' for play

### PR DESCRIPTION
**Pull request by Barry-Xu-2018**
https://github.com/ros2/rosbag2/blob/744cdf0420189b4435b9c8b2839899b1b80d23f5/ros2bag/ros2bag/verb/play.py#L51-L54

`nargs='+'` means that at least one topic is needed.   
But current description of this parameter means that no topic can be acceptable.

### CODE REVIEW

File: ros2bag/ros2bag/verb/play.py [hunk 1]
```diff
@@ -50,8 +50,7 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='rate at which to play back messages. Valid range > 0.0.')
         parser.add_argument(
             '--topics', type=str, default=[], nargs='+',
-            help='topics to replay, separated by space. If none specified, all topics will be '
-                 'replayed.')
+            help='List of topics to replay, separated by space. At least one topic is specified.')
```
Comment (fujitatomoya) [inline ros2bag/ros2bag/verb/play.py]:
```suggestion
            help='topics to replay, separated by space. At least one topic needs to be specified.')
```

**Review by fujitatomoya — APPROVED** [review]
I think it would better to adjust the description as you suggested instead of changing the behavior.

**Review by MichaelOrlov — APPROVED** [review]
LGTM.

**Comment by Barry-Xu-2018** [comment]
@MichaelOrlov Thanks for updating.

**Comment by MichaelOrlov** [comment]
@ros-pull-request-builder retest this please

File: ros2bag/ros2bag/verb/play.py [hunk 2]
```diff
@@ -50,8 +50,7 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='rate at which to play back messages. Valid range > 0.0.')
         parser.add_argument(
             '--topics', type=str, default=[], nargs='+',
-            help='topics to replay, separated by space. If none specified, all topics will be '
-                 'replayed.')
+            help='topics to replay, separated by space. At least one topic needs to be specified.')
```
Comment (emersonknapp) [inline ros2bag/ros2bag/verb/play.py]:
```suggestion
            help='Space-delimited list of topics to play.')
```

Bikeshed: I think the language should be like this. `ArgumentParser` will already validate input and provide informative `--help` given the `nargs`.

**Review by emersonknapp — APPROVED** [review]

**Comment by MichaelOrlov** [comment]
Gist: https://gist.githubusercontent.com/MichaelOrlov/cbf57ccb9c93494d2ed2b9b62b58c12d/raw/10122538790574f39e5c8a45ec0f575872aa7f76/ros2.repos
BUILD args:  --packages-above-and-dependencies ros2bag
TEST args:  --packages-above ros2bag
ROS Distro: rolling
Job: ci_launcher
ci_launcher ran: https://ci.ros2.org/job/ci_launcher/12665
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/19520/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/14038/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/20238/)

**Pull request merged**